beets/library.py

```diff
@@ -66,10 +66,6 @@ def __init__(self, field, pattern, fast=True, case_sensitive=None):
             case_sensitive = platform.system() != 'Windows'
         self.case_sensitive = case_sensitive
 
-        # Use a normalized-case pattern for case-insensitive matches.
-        if not case_sensitive:
-            pattern = pattern.lower()
-
```

Comment:
Ehm... yes, you're absolutely right. I must have overlooked this usage :open_mouth:...